Game/Config/DefaultEditorSpatialGDK.ini

```diff
@@ -8,52 +8,54 @@ TestSuiteGameStateBase=
 ; ClassName=include_path.h
 ; Leave empty if no includes required.
 
+/Game/ThirdPerson/TypebindingActor.TypebindingActor=
+
 ;PlayerController
-TestSuitePlayerController=TestSuitePlayerController.h
-TestSuitePlayerController=Camera/CameraAnim.h
-TestSuitePlayerController=Camera/CameraShake.h
-TestSuitePlayerController=GameFramework/HUD.h
-TestSuitePlayerController=GameFramework/LocalMessage.h
-TestSuitePlayerController=Particles/EmitterCameraLensEffectBase.h
+/Script/TestSuite.TestSuitePlayerController=TestSuitePlayerController.h
```

Comment:
Ugh, now that we've got the path in this file is so unsightly. We need to kill it with fire.

Reply:
I'm working on making it nicer;)